packages/react-positioning/src/middleware/coverTarget.ts

```diff
@@ -0,0 +1,30 @@
+import { Middleware } from '@floating-ui/dom';
+import { getBasePlacement } from '../utils/getBasePlacement';
+
+export function coverTarget(): Middleware {
+  return {
+    name: 'coverTarget',
+    fn: middlewareArguments => {
+      const { placement, rects, x, y } = middlewareArguments;
+      const basePlacement = getBasePlacement(placement);
```

Comment:
nit: Floating UI uses `side` as a word for this.

Reply:
I guess since it's internal it's quite ok to rename